src/Socket.cpp

```diff
@@ -30,7 +30,7 @@
 #ifdef HAVE_LIBSSL
 // Copypasted from
 // https://wiki.mozilla.org/Security/Server_Side_TLS#Intermediate_compatibility_.28default.29
```

Comment:
Let's just stick with Intermediate for out of the box compatibility - if someone wants to apply Modern they only have to add it to their configuration... out of the box I don't think Intermediate is bad, provided SSLv3 remains disabled in the protocols supported.